install/drain-cleaner/certmanager/000-Namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
From the Drain Cleaner files, you should include IMHO only the `openshift` folder. That should IMHO work for every AMQ Streams user and we do not need the cert-manager or raw kubernetes files.

Reply:
Works for me! Just removed!